src/azure-cli-core/azure/cli/core/style.py

```diff
@@ -72,3 +72,22 @@ def format_styled_text(styled_text):
     # Reset control sequence
     formatted_parts.append(Fore.RESET)
     return ''.join(formatted_parts)
+
+
+def get_styled_command(raw_command):
+    styled_command = []
+    argument_begins = False
+
+    for index, arg in enumerate(raw_command.split()):
+        spaced_arg = ' {}'.format(arg) if index > 0 else arg
+        style = Style.PRIMARY
+
+        if arg.startswith('-') and '=' not in arg:
```

Comment:
The logic of `=` handling is a little bit tricky. It works for `az config set a=b`, but it doesn't work for `az config unset output`. It  is the nature of positional argument that we can NOT decide whether an arg is 
- part of a command, or
- a positional argument 

until semantic analysis is involved. 

Let's keep it simple and treat it as part of the command by now.

Reply:
agree. @zhoxing-ms for awareness.